docs/manuals/source/DeveloperGuide/developerFaq.rst

```diff
@@ -0,0 +1,119 @@
+
+   $ cd mhvtl
+   $ make
+   $ sudo make install
+
+
+Now that mhvtl is installed, you can enable and start the mhvtl services
+
+.. code-block:: shell-session
+
+   $ sudo systemctl enable mhvtl-load-modules.service
+   $ sudo systemctl start mhvtl-load-modules.service
+   $ sudo systemctl enable mhvtl.target
+   $ sudo systemctl start mhvtl.target
+
+
+In order to check if your installation was done correctly, you can run :command:`lsscsi` (install it if you don't have it) or :command:`ls` in :file:`/dev/tape/by-id` which should give you somehting similar to the following output
+
+.. code-block:: shell-session
+
+   $ lsscsi
+   [1:0:0:0]    disk    ATA      SAMSUNG MZ7TY256 3L6Q  /dev/sda
+   [2:0:0:0]    mediumx STK      L700             0107  /dev/sch1
+   [2:0:1:0]    tape    IBM      ULT3580-TD8      0107  /dev/st1
+   [2:0:2:0]    tape    IBM      ULT3580-TD8      0107  /dev/st2
+   [2:0:3:0]    tape    IBM      ULT3580-TD8      0107  /dev/st0
+   [2:0:4:0]    tape    IBM      ULT3580-TD8      0107  /dev/st6
+   [2:0:8:0]    mediumx STK      L80              0107  /dev/sch0
+   [2:0:9:0]    tape    STK      T10000B          0107  /dev/st7
+   [2:0:10:0]   tape    STK      T10000B          0107  /dev/st4
+   [2:0:11:0]   tape    STK      T10000B          0107  /dev/st3
+   [2:0:12:0]   tape    STK      T10000B          0107  /dev/st5
+
+   $ ls /dev/tape/by-id/
+   scsi-350223344ab000100      scsi-350223344ab001000      scsi-XYZZY_A3
+   scsi-350223344ab000100-nst  scsi-350223344ab001000-nst  scsi-XYZZY_A3-nst
+   scsi-350223344ab000200      scsi-350223344ab001100      scsi-XYZZY_A4
+   scsi-350223344ab000200-nst  scsi-350223344ab001100-nst  scsi-XYZZY_A4-nst
+   scsi-350223344ab000300      scsi-SSTK_L700_XYZZY_A      scsi-XYZZY_B1
+   scsi-350223344ab000300-nst  scsi-SSTK_L80_XYZZY_B       scsi-XYZZY_B1-nst
+   scsi-350223344ab000400      scsi-XYZZY_A1               scsi-XYZZY_B2
+   scsi-350223344ab000400-nst  scsi-XYZZY_A1-nst           scsi-XYZZY_B2-nst
+   scsi-350223344ab000900      scsi-XYZZY_A2               scsi-XYZZY_B3
+   scsi-350223344ab000900-nst  scsi-XYZZY_A2-nst           scsi-XYZZY_B3-nst
+
+Now that the installtion is done, you will have to let bareos know that you are building tape and autochanger tests.
+To do so, you'll have to add the needed devices as cmake parameters to your exisiting cmake configuration:
+
+.. code-block:: shell-session
+
+   -Dchanger-device=/dev/tape/by-id/scsi-SSTK_L700_XYZZY_A
+   -Dtape-devices=/dev/tape/by-id/scsi-350223344ab000100-nst;/dev/tape/by-id/scsi-350223344ab000200-nst;/dev/tape/by-id/scsi-350223344ab000300-nst;/dev/tape/by-id/scsi-350223344ab000400-nst
+
+The selected devices are the ones used in our tests.
+
+In case of missing devices or other incoherences, a reboot of the machine can solve most of your issues.
+
+In case you update your kernel and kernel headers, you will have to rebuild and reinstall mhvtl, otherwise mhvtl will not work properly and bareos would not build.
```

Comment:
Maybe we can use stronger words.
```suggestion
For **any update** of kernel and kernel headers, you will have to rebuild and reinstall mhvtl, otherwise mhvtl kernel module loading will fail and command:`mhvtl` will not work properly and bareos will not build.
```